src/conf_mode/service_monitoring_telegraf.py

```diff
@@ -159,6 +162,11 @@ def verify(monitoring):
         if 'url' not in monitoring['splunk']:
             raise ConfigError(f'Monitoring splunk "url" is mandatory!')
 
+    # Verify Loki
+    if 'loki' in monitoring:
+        if 'url' not in monitoring['loki']:
+            raise ConfigError(f'Monitoring loki "url" is mandatory!')
```

Comment:
`url` should be uppercase

Reply:
The `url` is the missed option of the config, not the URL
Do we need URL here?

Reply:
Done!